extra_model/_cli.py

```diff
@@ -12,24 +12,30 @@
 
 
 OUTPUT_PATH = "/io"
+OUTPUT_FILENAME = "result.csv"
 EMBEDDINGS_PATH = "/embeddings"
 
 
 @click.command()
 @click.argument("input_path", type=Path)
 @click.argument("output_path", type=Path, default=OUTPUT_PATH)
+@click.argument("output_filename", type=Path, default=OUTPUT_FILENAME)
 @click.option("--debug", is_flag=True, help="Enable debug logging")
-def entrypoint(input_path: Path, output_path: Path, debug: bool = False) -> None:
+def entrypoint(
+    input_path: Path, output_path: Path, output_filename: Path, debug: bool = False
+) -> None:
     """Run the Extra algorithm for unsupervised topic extraction.
 
     INPUT_PATH is the path to the input parquet file with the user generated texts.
-    
+
     OUTPUT_PATH is the path to the output directory. Default is `/io`.
+
+    OUTPUT_FILENAME is the filename of the output file. Default is `result.csv`
```

Comment:
Mention here that we don't enforce `.csv`

Reply:
done